src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -920,8 +920,15 @@ def create_vm(cmd, vm_name, resource_group_name, image=None, size='Standard_DS1_
             'Please note that the default public IP used for VM creation will be changed from Basic to Standard '
             'in the future.')
 
-    subscription_id = get_subscription_id(cmd.cli_ctx)
+    # Breaking Change Warning, change image alias
+    if image:
+        if image == "Win2008R2SP1":
+            logger.warning(
+                '[Breaking Change] On April 04, 2023, the image deployed using "Win2008R2SP1" will reach its end of life.')
+        if image in ["RHEL", "Debian", "CentOS", "Flatcar"]:
+            logger.warning('Consider using the image alias including the version of the distribution you want to use.')
```

Comment:
Please give a example in the warning message
```suggestion
            logger.warning('Consider using the image alias including the version of the distribution you want to use. For example: please use Debian11 instead of Debian')
```